docs/guides/upgrading.rst

```diff
@@ -391,6 +391,26 @@ Plugin Messages
 
 Messages will no longer get the metadata 'msg' for newly created messages. This means you can not rely on that metadata to exist.
 
+Plugin Blog
+-----------
+
+The blog pages showing 'Mine' or 'Friends' listings of blogs have been changed to list all the blogs owned by the users instead of being created in the users container.
```

Comment:
Some people may not be that familiar with the concept of containers, so it would be clearer to just say that now they lists also the content created inside groups.